website/static/js/pages/sharing-page.js

```diff
@@ -40,3 +88,48 @@ $(function() {
         trigger: 'hover'
     });
 });
+
+var checkWindowWidth = function() {
+    if ($(window).width() <= 600) {
+        $('table.responsive-table-xxs tbody tr td:first-child').attr('role','button').attr('onclick', 'toggleExpand(this.parentElement)');
```

Comment:
Why is ```responsive-table-xss``` different than ```responsive-table-xs``` It looks like toggle should work the same in both of them and they share CSS, if they need to look slightly different then you should use classes like ```responsive-table table-xxs``` and ```responsive-table table-xs```. This will allow for using less code here and elsewhere.